docs/agents/srs_cg635.rst

```diff
@@ -0,0 +1,80 @@
+.. highlight:: rst
+
+.. _srs_cg635:
+
+====================
+SRS CG635 Agent
+====================
+
+The SRS CG635 Agent is an OCS Agent which retrieves data from the SRS CG635 clock
+via a Prologix GPIB interface.
+
+.. argparse::
+    :filename: ../socs/agents/srs_cg635/agent.py
+    :func: make_parser
+    :prog: python3 agent.py
+
+Configuration File Examples
+---------------------------
+
+Below are configuration examples for the ocs config file and for running the
+Agent in a docker container.
+
+OCS Site Config
+---------------
+
+To configure the SRS CG635 Agent we need to add an SRSCG635Agent
+block to our ocs configuration file. Here is an example configuration block
+using all of the available arguments::
+
+      {'agent-class': 'SRSCG635Agent',
+       'instance-id': 'srs-cg635',
+       'arguments': [['--ip-address', '10.10.10.10'],
+                     ['--gpib-slot', 23],
+                     ['--mode', 'acq']]},
+
+.. note::
+    The ``--ip-address`` argument should be the IP address of the Prologix GPIB interface.
+    The ``--gpib-slot`` argument should be the GPIB address set on the SRS CG635.
```

Comment:
For future reference I think it would be pretty useful to put here how to change the IP address of the prologix and the gpib address of the srs-cg635 since that was mildly annoying and required a little googling and random button smashing.